charts/operator-wandb/values.yaml

```diff
@@ -90,6 +90,22 @@ global:
     brokerPort: 9092
 
 ingress:
+  create: true
+  nameOverride: ""
+  defaultBackend: "app"
+  annotations: {}
+  labels: {}
+  tls: []
+  additionalHosts: []
+  class: ""
+  issuer:
+    create: false
+    provider: ""
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: support@wandb.com
+
+ingress_internal:
```

Comment:
We don't use `_` I think camelCase is that is being standardized on. 

Also, what is the ing

Reply:
Also, if this is for privatLink, maybe we should call this 
```
privatLinkIngress:
  create: false
...
```

or 
```
offlineIngress:
  create: false
...
```

We should probably document that this ingress is only needed for GCP/Azure/Bare metal?